NoteFind/Views/NoteCell.h

```diff
@@ -12,13 +12,17 @@
 #import "Parse/PFImageView.h"
 
 NS_ASSUME_NONNULL_BEGIN
+@protocol DataFailureDelegate
+- (void)showErrorMessage;
```

Comment:
`showErrorMessage` seems pretty generic, could we use a more specific name here?

Reply:
done!